configurator/src/capture.rs

```diff
@@ -218,18 +226,33 @@ impl Capturer {
         Ok(())
     }
 
-    fn distance_2d((x1, y1): (usize, usize), (x2, y2): (usize, usize)) -> f64 {
-        ((x1 as f64 - x2 as f64).powi(2) + (y1 as f64 - y2 as f64).powi(2)).sqrt()
+    fn distance_2d((x1, y1): &(usize, usize), (x2, y2): &(usize, usize)) -> f64 {
+        ((*x1 as f64 - *x2 as f64).powi(2) + (*y1 as f64 - *y2 as f64).powi(2)).sqrt()
     }
 
-    fn remove_outliers_by_2d_distance(
-        raw_points: Vec<WithConfidence<(usize, usize)>>,
-    ) -> Vec<WithConfidence<(usize, usize)>> {
+    fn distance_3d(a: &Vector3<f64>, b: &Vector3<f64>) -> f64 {
+        a.metric_distance(b)
+    }
+
+    pub fn reduce_confidence_of_outliers_by_3d_distance(
```

Comment:
Also this name is so long that it takes up half the line of code, the older version was better

Reply:
Fair point, but I'm still against using `remove` in the name of a method that doesn't remove anything. I've used `mark`.